cluster-autoscaler/cloudprovider/oci/nodepools/cache.go

```diff
@@ -61,10 +61,12 @@ func (c *nodePoolCache) rebuild(staticNodePools map[string]NodePool, maxGetNodep
 			}
 		}
 		if err != nil {
-			klog.Errorf("Failed to fetch the nodepool : %v", id)
-			return statusCode, err
```

Comment:
Now that we no longer return the status, what is the desired behavior of the statusCode return value?

The current behavior will just return whatever is the last statusCode. Is that what we should expect?

Reply:
right, I agree that **statusCode** became a bit redundant with this change.
It seems **statusCode** has been added to control refresh interval in case of GetNodePool api call returns 404. I think it was required to avoid Cluster Autoscaler doing too many refreshes when GetNodePool api call failed.

Since we suppress the api call failures now and do not return an error, ClusterAutoscaler will do the next refresh on its own interval. We can remove statusCode check from here as well as from the calling method.

https://github.com/kubernetes/autoscaler/blob/master/cluster-autoscaler/cloudprovider/oci/nodepools/oci_manager.go#L258-L262  

@trungng92 I will do a clean up if you agree.